docs/installation.rst

```diff
@@ -33,14 +33,7 @@ For using Inyoka and its dependencies you need a lot of python and developer fil
 
 .. code-block:: console
 
-    $  sudo apt-get install git nodejs-legacy libxml2-dev libxslt1-dev libzmq-dev zlib1g-dev libjpeg-dev uuid-dev libfreetype6-dev libpq-dev build-essential libpq-dev libffi-dev python3-dev
-
-You also need the JavaScript package manager npm:
-
-.. code-block:: console
-
-    $ sudo apt-get install npm
-
+    $  sudo apt install git npm libxml2-dev libxslt1-dev libzmq3-dev zlib1g-dev libjpeg-dev uuid-dev libfreetype6-dev libpq-dev build-essential libpq-dev libffi-dev python3-dev python3-venv
```

Comment:
I think we can also remove `libzmq3` here. As far as i remember it was only needed for XMPP. At least the docker container does not have it installed.